api/api.go

```diff
@@ -180,14 +184,40 @@ func (api *API) status(w http.ResponseWriter, req *http.Request) {
 			"buildDate": version.BuildDate,
 			"goVersion": version.GoVersion,
 		},
-		Uptime: api.uptime,
+		Uptime:     api.uptime,
+		MeshStatus: getStrippedDownMeshStatus(api),
 	}
 
 	api.mtx.RUnlock()
 
 	respond(w, status)
 }
 
+type strippedDownMeshStatus struct {
+	Connections []strippedDownLocalConnectionStatus `json:"connections"`
+}
+
+type strippedDownLocalConnectionStatus struct {
+	Address string `json:"address"`
+	State   string `json:"state"`
```

Comment:
Maybe describe in a comment what `Address` and `State` will look like. While I can imagine that `Address` will be an IPv4 address, an IPv6 address or a hostname I have no idea what `State` could be (maybe "healthy"?).